client/pages/login.tsx

```diff
@@ -21,16 +21,16 @@ export default () => {
   };
 
   return (
-    <div className='h-full grid grid-cols-2 gap-20'>
+    <div className='h-full grid grid-cols-2 gap-20 mx-20'>
       <div className='flex flex-col justify-center'>
-        <h2 className='text-4xl'>BC Air Access Program Application</h2>
+        <h2 className='text-4xl'>BC Active Transportation Application</h2>
         <p>
-          The BC Air Access Program (BCAAP) is an application-based program that provides capital
-          cost-sharing contributions to aviation infrastructure projects. This includes facility
-          master plans, greenhouse gas audits or baselining, and GPS approaches. Support to the
-          aviation sector is critical to helping BC address its responsibilities concerning medevac,
-          wildfire suppression, emergency response, access to remote (often Indigenous) communities,
-          clean transportation, tourism, and economic development.
+          The BC Active Transportation (BCAT) program is an application-based program that provides
```

Comment:
I feel like this description needs to be specific to BCAT

Reply:
Yup, I left a `TO-DO` in the PR description as well as the ticket saying we need updated texts regarding BCAAP descriptions, but seeing as we don't have those yet I just removed BCAAP with BCAT instead for now.

Reply:
Maybe comment it out for now instead of renaming to BCAT? This way it won't be overlooked later